src/webextension/unlock/components/app.css

```diff
@@ -13,9 +13,13 @@ body {
   -moz-user-select: none;
 }
 
+.unlock {
+  height: 360px;
```

Comment:
Let's move this up to the `body` element, where `width` is defined.

Reply:
Defining the height on the body instead of either the <main /> or <article /> will not create the desired effect (unless we also define those as being height: 100%). If that is preferred I can make the change, the same applies to the popup.

Reply:
Hmm, that might be a bug in our `<Panel/>` CSS. I'll take a look.

Reply:
I don't think it's a bug, simply that the footer of the panel is fixed to the bottom of the `<article />`, which in both instances has a wrapper which isn't set to 100% height.

Reply:
Oh wait, I understand. Let's do something similar to the management UI:

https://github.com/mozilla-lockbox/lockbox-extension/blob/b640724d02b0dac86234c00099ace8e53507bd46/src/webextension/list/manage/components/app.css#L18-L21

I think that'll make it extra-clear that the dimensions should apply to the size of the entire popup window.